home/text_script.asm

```diff
@@ -66,12 +66,12 @@ DisplayTextID::
 
 ; check first byte of text for special cases
 
-dict2: MACRO
+MACRO dict2
 	cp \1
 	jr nz, .not\@
 	\2
 	jr AfterDisplayingTextID
-.not\@
+	.not\@
```

Comment:
I acknowledge everything done here is consistent, but I wonder if labels like this would still look better unindented.
Not a big deal either way though.

Reply:
@Rangi42 what's your opinion?

Reply:
I could also see the counter of leaving the labels indented.. and indenting the code an extra time.

Reply:
IMO labels should be indented one level less than their code. For example:

```
    jr z, .done
    ld a, 42
.done

MACRO mac
    jr z, .foo\@
    ld a, 42
.foo\@
ENDM

MACRO mac2
    IF bar
        jr z, .foo\@
        ld a, 42
    .foo\@
    ENDC
ENDM
```

Reply:
Alright; give me like an hour and I'll fix this across the repos.